pkg/kn/plugin/manager_test.go

```diff
@@ -301,6 +301,33 @@ func TestPluginList(t *testing.T) {
 	}
 }
 
+func TestNoSlashInPlugin(t *testing.T) {
+	ctx := setup(t)
+	defer cleanup(t, ctx)
+
+	// Prepare PATH
+	tmpPathDir, cleanupFunc := preparePathDirectory(t)
+	defer cleanupFunc()
+
+	fmt.Println(tmpPathDir)
+
+	createTestPluginInDirectory(t, "kn-path-test", tmpPathDir)
+	pluginCommands := []string{"path", "test", "/withslash"}
```

Comment:
The test should also use a `os.PathSeparator` otherwise it will fail on Windows.

Also I would suggest to use a loop and multiple tests, e.g.

```
path test /withslash
path test /withslash extra arg
path test ext\ra  (using something like `os.PathSeparator == '/' ? "\\" : "/"`)
```